bot/codex.py

```
@@ -0,0 +1,483 @@
+        servers = data.get("mcp_servers") or {}
+        if not isinstance(servers, dict):
+            return args
+        for server_id in servers:
+            if server_id != "comfytv":
+                args += ["-c", f"mcp_servers.{server_id}.enabled=false"]
+        return args
+
+    def _write_attachment(self, home: str, data: str, media_type: str) -> Optional[str]:
+        try:
+            raw = base64.b64decode(data)
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)